modules/cluster-logging-release-notes-5.2.z.adoc

```diff
@@ -0,0 +1,234 @@
+//Z-stream Release Notes by Version
+
+[id="cluster-logging-release-notes-5-2-6"]
+= OpenShift Logging 5.2.6
+
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0230[RHSA-2022:0230 OpenShift Logging Bug Fix Release 5.2.6]
+
+[id="openshift-logging-5-2-6-bug-fixes"]
+== Bug fixes
+* Before this update, the release did not include a filter change which caused fluentd to crash. With this update, the missing filter has been corrected. (link:https://issues.redhat.com/browse/LOG-2104[LOG-2104])
+
+* This update changes the log4j dependency to 2.17.1 to resolve link:https://access.redhat.com/security/cve/CVE-2021-44832[CVE-2021-44832].(link:https://issues.redhat.com/browse/LOG-2101[LOG-2101])
+
+[id="openshift-logging-5-2-6-CVEs"]
+== CVEs
+.Click to expand CVEs
+[%collapsible]
+====
+* link:https://access.redhat.com/security/cve/CVE-2021-27292[CVE-2021-27292]
+** link:https://bugzilla.redhat.com/show_bug.cgi?id=1940613[BZ-1940613]
+* link:https://access.redhat.com/security/cve/CVE-2021-44832[CVE-2021-44832]
+** link:https://bugzilla.redhat.com/show_bug.cgi?id=2035951[BZ-2035951]
+====
+
+[id="cluster-logging-release-notes-5-2-5"]
+= OpenShift Logging 5.2.5
+
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0043[RHSA-2022:0043 OpenShift Logging Bug Fix Release 5.2.5]
+
+[id="openshift-logging-5-2-5-bug-fixes"]
+== Bug fixes
+* Before this update, Elasticsearch rejected logs from the eventrouter due to a parsing error. This update changes the data model to resolve the parsing error. However, as a result, existing indices will cause an warning/error within Kibana. The field ("kubernetes.event.metadata.resourceVersion") will cause an error until the removal or reindexing of existing indices.
+
+If this field is not in use in Kibana, then you can ignore the error message. For those with a delete policy, that policy will eventually remove old indices. When that occurs, the message will no longer generate. Those without a delete policy in place will need to manually reindex in order to remove the message.(link:https://issues.redhat.com/browse/LOG-2087[LOG-2087])
```

Comment:
```suggestion
If the `kubernetes.event.metadata.resourceVersion` field is not in use in Kibana, you can ignore the error messages. If you have a delete policy, it eventually removes old indices, which stops the error messages. If you don't have a delete policy, you can manually delete an old index to stop the error messages.(link:https://issues.redhat.com/browse/LOG-2087[LOG-2087])
```
 - It sounds like, if the `kubernetes.event.metadata.resourceVersion` field is in use, must the user take action? Or is it optional? Maybe make this clearer.